pyjob/slurm.py

```diff
@@ -56,6 +56,8 @@ def info(self):
     def close(self):
         """Close this :obj:`~pyjob.slurm.SlurmTask` after completion"""
         self.wait()
+        if self.cleanup_files and self.runscript is not None:
+            self.runscript.cleanup()
```

Comment:
I think we should move the def close definition to the base class, then only override in `local.py`. seems unnecessary to re-define all this every time

Reply:
I moved this to `pyjob.ClusterTask`, which means it wasn't necessary to override `pyjob.LocalTask` (it inherits from `pyjob.Task` not from `pyjob.ClusterTask`)